.github/workflows/publish.yml

```diff
@@ -169,9 +169,21 @@ jobs:
       - build-ami
     if: always() && github.head_ref == 'update_flake_lock_action'
     steps:
-      - name: Notify
+      - name: Notify Discord
         uses: rjstone/discord-webhook-notify@v1
         with:
           severity: info
           details: NixOS AMIs Update - ${{ needs.build-ami.result }}
           webhookUrl: ${{ secrets.DISCORD_WEBHOOK_URL }}
+      - name: Notify Slack
+        uses: slackapi/slack-github-action@v2.1.0
+        with:
+          webhook: ${{ secrets.SLACK_WEBHOOK_URL }}
+          webhook-type: incoming-webhook
+          payload: |
+            text: "*NixOS AMI Update*: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
+            blocks:
+              - type: "section"
+                text:
+                  type: "mrkdwn"
```

Comment:
I'm so glad we saved the 2 characters here lmao (I'm sure this is what Slack said to do but... amazing)

Reply:
E F F I C I E N T